pkg/operator2/idp.go

```diff
@@ -216,7 +233,11 @@ func (c *authOperator) convertProviderConfigToIDPData(providerConfig *configv1.I
 	return data, nil
 }
 
-func (c *authOperator) discoverOpenIDURLs(issuer, key string, ca configv1.ConfigMapNameReference) (*osinv1.OpenIDURLs, error) {
+// discoverOpenIDConfig retrieves basic information about an OIDC server with hostname
```

Comment:
nit: This comment does not appear to match the function signature, which lacks a boolean return value.